.github/workflows/moodle-ci.yml

```diff
@@ -0,0 +1,108 @@
+name: Moodle Plugin CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_USER: 'postgres'
+          POSTGRES_HOST_AUTH_METHOD: 'trust'
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 3
+
+      mariadb:
+        image: mariadb:10
+        env:
+          MYSQL_USER: 'root'
+          MYSQL_ALLOW_EMPTY_PASSWORD: "true"
+          MYSQL_CHARACTER_SET_SERVER: "utf8mb4"
+          MYSQL_COLLATION_SERVER: "utf8mb4_unicode_ci"
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries 3
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['8.1', '8.2']
```

Comment:
![Screenshot from 2024-04-18 09-40-45](https://github.com/owncloud/moodle-repository_ocis/assets/54478846/2e60e7d6-bc35-4dc5-b40d-d47292cefa8d)

should we run our tests on PHP 8.1 ? I think php8.2 and 8.3 will be better?

Reply:
we should run the tests with 8.1, because that is what we claim in the readme.
We can add 8.3 if the software works with it

Reply:
the 8.1 and 8.2 should be fine